frameworks/zuul.js

```diff
@@ -198,11 +197,11 @@ ZuulReporter.prototype.done = function (err) {
 
   // add coverage tab content
   if (window.__coverage__) {
-    var coverage_tab = document.getElementById('code-coverage-tab')
-    coverage_tab.innerHTML = '<iframe frameborder="0" src="/__zuul/coverage"></iframe>'
+    var coverageTab = document.getElementById('code-coverage-tab')
+    coverageTab.innerHTML = '<iframe frameborder="0" src="/__zuul/coverage"></iframe>'
   }
 
-  post_message({
+  postMessage({
```

Comment:
Maybe rename to avoid confusion with `window.postMessage`

Reply:
`bufferMessage`?

Reply:
That'd work. Could also replace these calls with `messageBus.push`. Take your pick :)